sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -434,7 +434,6 @@ abstract class ScrollableListPainter extends Painter {
   double _scrollOffset = 0.0;
   void set scrollOffset (double value) {
     assert(value != null);
-    assert(value >= 0.0 && value <= 1.0);
```

Comment:
This is an excellent fix. I'm very confused about how it has survived there without my noticing it. Did I accidentally throw a no-asserts switch for my local environment?